### PR TITLE
Wrong Spellings in PULL_REQUEST_TEMPLATE

### PR DESCRIPTION
**Pull request by DhruvamSharma**
Changes: Wrong spellings in line 3: ther.
Wrong Indentation in line 3: And starting just after a full stop.
Wrong grammar in line 5: for used instead of-of.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Fixed #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]
 
-Changes: [Mention the files changed in ther PR.Add here what changes were made in this issue and if possible provide links(Deploy/Preview Link).]
+Changes: [Mention the files changed in their PR. Add here what changes were made in this issue and if possible provide links(Deploy/Preview Link).]
```
Comment (isuruAb) [inline .github/PULL_REQUEST_TEMPLATE]: should be "the"?

**Comment by abishekvashok** [comment]
Can you correct as requested please?

File: .github/PULL_REQUEST_TEMPLATE (hunk 1 above)
Comment (abishekvashok) [reply .github/PULL_REQUEST_TEMPLATE]: I guess so.

**Comment by DhruvamSharma** [comment]
@isuruAb  Done with the changes.

**Pull request merged**